asm/vm/asm_vm

```diff
@@ -1001,6 +1001,12 @@ EOF
 # Generate GCE service proxy definition for the instance template
 gen_gce_service_proxy() {
   local PROXY
+
+  local ASM_META_REVISION="${WORKLOAD_ASM_REVISION}"
+  if [ "${ASM_META_REVISION}" == "default" ]; then
```

Comment:
What if the user has a revision called default? Maybe we should have a _USE_CI toggle for that.

Reply:
If their revision is called `default` there are bigger issues... and they probably want this to get hit?